File: src/mantle/internal/crash/CallableSuppConfig.scala
package mantle.internal.crash

import cpw.mods.fml.common.ICrashCallable

/**
 * Crash Callable for supported environments.
 *
 * @author Sunstrike <sunstrike@azurenode.net>
 */
class CallableSuppConfig extends ICrashCallable {

  def getLabel = "Mantle Env"

  def call = "Environment healthy."

}

File: src/mantle/internal/crash/CallableSuppConfig.java
package mantle.internal.crash;

import cpw.mods.fml.common.ICrashCallable;

/**
 * Crash Callable for supported environments.
 *
 * @author Sunstrike <sunstrike@azurenode.net>
 */
public class CallableSuppConfig implements ICrashCallable
{

    @Override
    public String call () throws Exception
    {
        return "Environment healthy.";
    }

    @Override
    public String getLabel ()
    {
        return "Mantle Env";
    }

}
